pkg/scaffold/v1/resource/resource.go

```diff
@@ -54,13 +54,13 @@ type Resource struct {
 
 // Validate checks the Resource values to make sure they are valid.
 func (r *Resource) Validate() error {
-	if len(r.Group) == 0 {
+	if len(r.Group) == 0 || r.Group == "--version" || r.Group == "--kind" {
```

Comment:
This makes the code ugly.
Can we leverage https://godoc.org/github.com/spf13/cobra#Command.MarkFlagRequired?

Reply:
Sure .. very better, tks for point it.